sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -42,20 +42,22 @@
 
 
 class DatabaseProxy(object):
-    """
-    An interface to interact with a specific database.
-    This class should not be instantiated directly, use :func:`CosmosClient.get_database_client` method.
+    """An interface to interact with a specific database.
+
+    This class should not be instantiated directly. Instead use the
+    :func:`CosmosClient.get_database_client` method.
 
     A database contains one or more containers, each of which can contain items,
     stored procedures, triggers, and user-defined functions.
 
     A database can also have associated users, each of which configured with
```

Comment:
each of which [is, can be] configured....

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/8607/commits/c14bb0d8adcb5e63ac1d6a2fab3819ccca50707e